views/jira-configuration.hbs

```diff
@@ -1,348 +1,254 @@
 <html>
-  <head>
```

Comment:
I see a lot of git diff for formatting only here. Do you mind update it so that here's no change in these whitespaces?
![image](https://user-images.githubusercontent.com/105693507/176830287-e5502c27-e22d-4b96-ab2e-de5f457fd65b.png)